manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+PackageLocale: en-US
+Publisher: F-Secure

```

```suggestion
Publisher: F-Secure Corporation
```
This should match control panel